src/net_processing.cpp

```diff
@@ -2051,6 +2053,49 @@ static bool PrepareBlockFilterRequest(CNode* pfrom, const CChainParams& chain_pa
     return true;
 }
 
+/**
+ * Handle a cfilters request.
+ *
+ * May disconnect from the peer in the case of a bad request.
+ *
+ * @param[in]   pfrom           The peer that we received the request from
+ * @param[in]   vRecv           The raw message received
+ * @param[in]   chain_params    Chain parameters
+ * @param[in]   connman         Pointer to the connection manager
+ */
+static void ProcessGetCFilters(CNode* pfrom, CDataStream& vRecv, const CChainParams& chain_params,
```

Comment:
nit: since `pfrom` and `connman` are assumed, how about pass by reference?

Reply:
You're the second reviewer to ask for this so I've now made the change (as well as to `ProcessGetCFHeaders()` and `ProcessGetCFCheckPt`).

See also https://github.com/bitcoin/bitcoin/pull/19010#discussion_r429175250.

Reply:
Thanks, IMO eliminating the null case is a worthwhile reason to break convention.